sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj

```diff
@@ -3,18 +3,13 @@
   <PropertyGroup>
     <Description>This is the Microsoft Azure Key Vault Certificates client library</Description>
     <AssemblyTitle>Microsoft Azure.Security.KeyVault.Certificates client library</AssemblyTitle>
-    <Version>4.0.0-preview.7</Version>
+    <Version>4.0.0-preview.8</Version>
     <PackageTags>Microsoft Azure Key Vault Certificates;$(PackageCommonTags)</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-      - Moved Cancel and Delete certificate operation methods from CertificateClient to CertificateOperation.
-      - CertificateClient.DeleteCertificate has been renamed to CertificateClient.StartDeleteCertificate and now returns a DeleteCertificateOperation to track this long-running operation.
-      - CertificateClient.RecoverDeletedCertificate has been renamed to CertificateClient.StartRecoverDeletedCertificate and now returns a RecoverDeletedCertificateOperation to track this long-running operation.
-      - CertificateIssuer.Administrators has been renamed to CertificateIssuer.AdministratorContacts.
-      - CertificateKeyType.Oct has been removed.
-      - The SubjectAlternativeNames class now allows you to add multiple types of subject alternative names using any of the DnsNames, Emails, and UserPrincipalNames collection properties.
-
-      See https://github.com/Azure/azure-sdk-for-net/tree/Azure.Security.KeyVault.Certificates_4.0.0-preview.7/sdk/keyvault/Azure.Security.KeyVault.Certificates/ChangeLog.md for more detailed changes.
+      A small number of parameters were renamed or made required to better align with other languages and to reflect requirements of the Azure Key Vault service.
+
+      See https://github.com/Azure/azure-sdk-for-net/tree/Azure.Security.KeyVault.Certificates_4.0.0-preview.8/sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md for more detailed changes.
```

Comment:
Does CHANGELOG need to be updated as part of this PR?

Reply:
I keep it up to date as I make changes. It's ready.